server/graphql/index.ts

```diff
@@ -85,6 +85,31 @@ export const rootValue = {
           space
         };
       });
+    },
+    spaces: (
+      parent,
+      { all = false, first = 10, skip = 0, id, spaces = [] },
```

Comment:
What's the need of `all`? If you want all the space we should just have to query this
```
spaces (first: 1000) { name }
```